client/zweb/deployer/index.js

```diff
@@ -3,11 +3,16 @@ const fs = require('fs');
 const utils = require('#utils');
 const ethUtil = require('ethereumjs-util');
 
+// TODO: direct import cause fails in some docker scripts
+let storage;
+
 class Deployer {
     constructor(ctx) {
         this.ctx = ctx;
         this.config = this.ctx.config.deployer;
         this.cache_uploaded = {}; // todo: unused? either remove or use
+        const _storage = require("../../../client/storage/index-new.js");
+        storage = _storage;
```

Comment:
Why do we need this intermediary variable `_storage`?
And also, why do you `require` the file in constructor. Is there a circular dependency?

Reply:
I think there was an issue with require, but I'll re-check

Reply:
Yep, if I move require to the top, e2e fails. You can see that other relative requires are also done dynamically.

Reply:
Noted on circular dependency. But why do we use the `_storage` variable then? Why not to write `storage = require("../../../client/storage/index-new.js");`?

Reply:
But I've already updated it